README.md

```diff
@@ -236,6 +237,35 @@ end
 
 You can access user model through authenticatable.
 
+### Claiming tokens
+
+Opt-in for marking tokens as `claimed` so that they can't be used again.
+
+config/initializers/passwordless.rb
+
+```ruby
+# Default is `false`
+Passwordless.claim_token_after_sign_in = true
+```
+
+#### Upgrading an existing Rails app
```

Comment:
Very nice to include this bit! However I think we should move it into a `<details>` element ([examples](https://gist.github.com/ericclemmons/b146fe5da72ca1f706b2ef72a20ac39d)) to avoid distracting people who are not going to use the feature.

We can probably also write a shorter description. Something like ...

> The simplest way to update your sessions table is with a single migration: 

[example]

> Be aware that this will lock your database while migrating. For a _higher availability_ approach, see this [example gist](#).